azure-pipelines.yml

```diff
@@ -6,6 +6,43 @@ trigger:
 
 jobs:
 
+- job: 'Test_bare_linux'
+
+  pool:
+    vmImage: 'ubuntu-16.04'
+  strategy:
+    matrix:
+      Python35:
+        python.version: '3.5'
+      Python36:
+        python.version: '3.6'
+      Python37:
+        python.version: '3.7'
+
+  steps:
+  - task: UsePythonVersion@0
```

Comment:
Just for my own information, where does the choice of @ versions come from here and below?

Reply:
I'm not sure that I understand the scope of the question, so let me know if this doesn't actually answer it. It's probably fine to just choose 1 python version for this bare environment test, but it's easy enough to add the matrix option so I figured why not. In general, we want to test pvlib against all supported Python versions. This is stated in the readme and installation docs as "3.5 and above". 3.8 just came out, so I will look into adding that in a few weeks. I'm pretty confident that pvlib itself will not have any issues with 3.8, but it often takes a few weeks for the dependencies to release new wheels and/or Anaconda to add compiled packages.

Reply:
More specifically, how do you know what the @0, @1, and @3 suffixes refer to?

Reply:
Oh, I don’t understand that part of the spec. I’m just following the docs here https://docs.microsoft.com/en-us/azure/devops/pipelines/ecosystems/python?view=azure-devops

Reply:
Specifies the task version number: https://docs.microsoft.com/en-us/azure/devops/pipelines/process/tasks?view=azure-devops&tabs=yaml#task-versions

Reply:
Oh I think I get it now. Microsoft supplies those task versions so they don’t break people’s pipelines when they make changes.